spec.bs

```diff
@@ -488,7 +505,7 @@ Issue(190): Define "cryptographic procedures to obtain a list of unmasked tokens
 
 The details for servers implementing this protocol can be found in [[!ISSUER-PROTOCOL]].
 
-<h3 id="modification-to-http-fetch-steps-header">Modification to HTTP fetch steps</h3>
+<h3 id="modification-to-http-fetch-steps-header-issue">Modification to HTTP fetch steps</h3>
```

Comment:
I think we should merge both issue and redemption fetch monkeypatches under a single header and move it up into https://wicg.github.io/trust-token-api/#fetch-integration. Not sure why I didn't originally put it there :)